### PR TITLE
Added ltfschoen

### PR DESCRIPTION
**Pull request by ltfschoen**
https://kusama.subscan.io/extrinsic/24958619-2

### CODE REVIEW

File: registry/kusama.json [hunk 1]
```diff
@@ -278,5 +278,13 @@
         "shortDescription": "Kusama validator.",
         "longDescription": "Japan",
         "isOrganization": false
+    },
+    {
+        "address": "DDCNPp8oeYBcBM44b32iSse4t4yfTnDJbjQxohF59Fo23EF",
+        "name": "Luke Schoen (ltfschoen)",
+        "image": "https://raw.githubusercontent.com/ltfschoen/opengov-delegate-registry/master/images/ltfschoen-image.png",
```
Comment (stepanLav) [inline registry/kusama.json:285]: Hello!
Thank you for your request, based on our rules all data should be in /novasamatech/opengov-delegate-registry/ , please, change it in order to get image from novasama repo

**Comment by ltfschoen** [comment]
https://kusama.subscan.io/extrinsic/24958619-2

File: registry/kusama.json (hunk 1 above)
Comment (ltfschoen) [inline registry/kusama.json:285]:
```suggestion
        "image": "https://raw.githubusercontent.com/novasamatech/opengov-delegate-registry/master/images/ltfschoen-image.png",
```

Comment (ltfschoen) [reply registry/kusama.json]: thanks for the feedback, i've fixed that in this commit https://github.com/novasamatech/opengov-delegate-registry/pull/109/commits/13ec6d0be527f668a7ac42d69d42ef953a91cfeb

File: registry/kusama.json [hunk 2]
```diff
@@ -278,5 +278,13 @@
         "shortDescription": "Kusama validator.",
         "longDescription": "Japan",
         "isOrganization": false
+    },
+    {
+        "address": "DDCNPp8oeYBcBM44b32iSse4t4yfTnDJbjQxohF59Fo23EF",
+        "name": "Luke Schoen (ltfschoen)",
```
Comment (ltfschoen) [inline registry/kusama.json:284]:
```suggestion
        "name": "Luke Schoen @ltfschoen",
```

File: registry/kusama.json [hunk 3]
```diff
@@ -278,5 +278,13 @@
         "shortDescription": "Kusama validator.",
         "longDescription": "Japan",
         "isOrganization": false
+    },
+    {
+        "address": "DDCNPp8oeYBcBM44b32iSse4t4yfTnDJbjQxohF59Fo23EF",
+        "name": "Luke Schoen @ltfschoen",
```
Comment (ltfschoen) [inline registry/kusama.json:284]:
```suggestion
        "name": "Luke Schoen @ltfschoen | Clawbird DAO",
```

**Review by stepanLav — APPROVED** [review]

**Comment by ltfschoen** [comment]
@stepanLav I just installed latest v8.7.0 dated 7 October 2025 of Nova Polkadot Wallet on Android Play Store but I still can't see myself as a delegate in the list to choose from even though this PR was merged

**Pull request merged**